packages/snap/src/rpc/signMessage.ts

```diff
@@ -0,0 +1,14 @@
+import {Message, SignedMessage, transactionSign} from "@zondax/filecoin-signing-tools/js";
+import {Wallet} from "../interfaces";
+import {getKeyPair} from "../filecoin/account";
+import {showConfirmationDialog} from "../util/confirmation";
+
+export async function signMessage(wallet: Wallet, message: Message): Promise<SignedMessage> {
+  const confirmation =
+      await showConfirmationDialog(wallet, "Do you want to sign message with your private key?");
```

Comment:
@mpetrun5 Not sure if this is the best message, what do you think? Maybe I can add some `message` details, but then I should add all properties and that seems to me as too much info on prompt.